extras/ioc_hash_scan.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+#
+##Add the hashes to the blob at src/ioc_scan/ioc_scanner.py
+#
+#
+#  The filename specified in the first argument
+# (instance-list-file) should contain a list of instance id strings, one per line.
+#
+# Usage: ./ioc_hash_scan.sh instance-list-file <AWS_PROFILE>
+#
+# Must have AWS_CREDENTIAL_FILE and AWS_REGION exported to environmental varialble
+#
+# Example - Run:
+# export AWS_CREDENTIALS_FILE="$HOME/.aws/credentials" >> "$HOME/.bashrc"
+# export AWS_REGION="us-east-1" >> "$HOME/.bashrc"
+# Then reload your shell, like
+# source ~/.bashrc
+#
+# This script assumes that it exists in the ioc-scanner/extras/ directory.
+# If this is different, please edit the variable $pydir to point to
+# the directory containing ioc_scanner.py
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Directory path of the ioc_scanner.py
+pydir="../src/ioc_scan/"
+
+if [ $# -eq 0 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ] || [ $# -lt 2 ]; then
+  echo Usage: "$0" instance-list-file \<AWS_PROFILE\>
+  exit 1
+fi
+
+# Check if instance list file exists.
+if [ ! -f "$1" ]; then
+  echo Instance List file "$1" does not exist - exiting.
+  exit 1
+fi
+
+# Read instance id strings from file.  [[ -n "$line" ]] handles the case where
+# the last line doesn't end with a newline.
+serverList=()
+while IFS= read -r line || [[ -n "$line" ]]; do
+  serverList+=("$line")
+done < "$1"
+
+if [ ${#serverList[@]} -eq 0 ]; then
+  echo No Instances found in "$1" - exiting.
```

Comment:
Remove unneeded capitalization:
```suggestion
  echo No instances found in "$1" - exiting.
```